docs/shared/blockquote-stop.html

```diff
@@ -0,0 +1 @@
+<div class="special-quote special-quote-stop fa fa-ban"></div>
```

Comment:
I thought exactly the same! Let me try it out and provide you a screenshot here...

Reply:
Hm, changing the background color is not easy because mdbook has multiple themes and we would need to take that into account as each theme has a different font color, etc.

Changing the border color has a pretty dramatic effect though, hah, look at this:

<img width="815" alt="Screenshot 2021-04-30 at 10 49 20" src="https://user-images.githubusercontent.com/25176023/116671951-dbf49480-a9a1-11eb-94ef-d64c6664a3d7.png">
<img width="791" alt="Screenshot 2021-04-30 at 10 51 36" src="https://user-images.githubusercontent.com/25176023/116672093-0d6d6000-a9a2-11eb-87b0-9f36984ae1b2.png">

Let me know if you think, too dramatic. Personally, I am fan 😄

Reply:
I love drama. +1